cardano-node/README.md

```diff
@@ -206,14 +206,22 @@ transaction, in its raw wire format (see GenTx for Byron transactions).
 The canned `scripts/submit-tx.sh` script will submit the supplied transaction to a testnet
 launched by `scripts/shelley-testnet*.sh` family of scripts.
 
-### Issuing genesis UTxO expenditure Tx
+### Issuing UTxO expenditure (genesis and regular)
 
-To make a transaction spending genesis UTxO, you can either use the `issue-genesis-utxo-expenditure`
-subcommand directly, or, again use a canned script that will make a transaction tailored
-for the aforementioned testnet cluster -- `scripts/issue-genesis-utxo-expenditure.sh`.
+To make a transaction spending UTxO, you can either use the:
```

Comment:
We don't need to do this in this PR, but I think we should migrate all instructions wrt scripts to `SCRIPTS.md` and just leave a short section in the top level `README.md` pointing to this.

Reply:
I think we can address that in a future PR, what do you think?